staging/publishing/import-restrictions.yaml

```diff
@@ -207,6 +207,7 @@
   allowedImports:
   - k8s.io/api
   - k8s.io/apimachinery
+  - k8s.io/apiserver
```

Comment:
curious what required this one

Reply:
if this link is easy to snip, I'd recommend doing so. apiserver pulls in a lot of transitive dependencies

Reply:
All forms of https://github.com/kubernetes/kubernetes/blob/master/pkg/cloudprovider/providers/gce/gce_disks.go#L149. Essentially checking if certain cloud features for disk are enabled. 
Will look at snipping it tomorrow.

Reply:
https://github.com/kubernetes/kubernetes/pull/74021 relocates the feature gate package out of k8s.io/apiserver for just this purpose.

Reply:
component-base is a much more reasonable dep

Reply:
> component-base is a much more reasonable dep

Agreed. @andrewsykim Do you have time to push #74021 forward? Would you like help?

Reply:
Sorry, that one slipped my radar. I can get to it today :)

Reply:
actually, looks like GCERegionalPersistentDisk is the only feature checked, and that went GA in 1.13.

can we lock that feature on and remove all uses of it in gce? if so, I think we can drop the utilfeature check entirely and skip waiting on https://github.com/kubernetes/kubernetes/pull/74021

Reply:
removed use of the feature gate in https://github.com/kubernetes/kubernetes/pull/77412

Reply:
Merged, this should be in the clear once it rebases